### ISSUE TITLE
Question about dir(i,j) matrix in equation(8)

### ISSUE THREAD

**Issue by yinyueqin** [linjieli222/VQA_ReGAT#14]
Sincerely thank for your sharing codes.

However, I found the dir matrix0  and dir[1] are actually the same in spatial relation.  That is the condensed_adj_matrix in the `condensed_adj_matrix = torch.sum(input_adj_matrix, dim=-1)`. If the condensed_adj_matrix[0][i][j]=1, then condensed_adj_matrix[1][i][j] = 1 too which means bbx[0] and bbx[1] has a spatial relation(edge). So I feel very confused about it~ 
![image](https://user-images.githubusercontent.com/34408423/77737253-9223d700-7048-11ea-9db2-9da80c146002.png)

**Comment by linjieli222** [comment]
Thank you for your question and sorry for the late reply.

For spatial relation, if there is an edge i->j, then there would be an edge j->i.  For example, if bbox i is inside bbox j, then bbox j covers bbox i. So the spatial graph is symmetric in terms of edges (disregards of the edge labels).

**Comment by alice-cool** [comment]
Where to consider the edge labels???? I only get the adj edge matrix...
